contracts/EIP5058/ERC721Lockable.sol

```diff
@@ -51,6 +51,14 @@ abstract contract ERC721Lockable is Context, ERC721, IERC721Lockable {
      */
     function lockerOf(uint256 tokenId) public view virtual override returns (address) {
         require(_exists(tokenId), "ERC721L: locker query for nonexistent token");
+        // NOTE
+        //
+        // why empty address?
+        //
+        // if return empty address it will be impossible to do `unlockFrom` anymore
+        // which
+        //  - will throw a wrong error message
+        //  - only delete the kv in the mapping
```

Comment:
side effects to `lockFrom`, throw `ERC721L: unlock caller is not lock operator` when the operator is actual the operator